content/blog/integrating-istio-and-spire.md

```diff
@@ -0,0 +1,463 @@
+
+![](/img/spire.png)
+
+SPIRE can be configured as a source of cryptographic identities for Istio workloads through an integration with Envoy’s SDS (Secret Discovery Service) API. 
+This integration with SPIRE provides flexible attestation options not available with the default Istio identity management while harnessing Istio’s powerful service management. 
+
+![](/img/picture1.jpg)
+
+Later this SPIRE issued certificates or identities to workloads or services can be used for communication between different trust domains or between two different clusters also. 
+
+Use the steps in this blog to install Istio and SPIRE on the same cluster, then deploy a sample application using SPIRE-issued identities.  
+
+### Step 1: Creating your own cluster
+
+**1﻿.1**  Go to [HPE Greenlake ](https://client.greenlake.hpe.com/session/hub/choose)client page. 
+
+**1.2**  Login to your tenant in HPE GreenLake Central  and navigate to HPE GreenLake for Private Cloud Enterprise dashboard. Click on Containers to launch into the containers dashboard. 
+
+![](/img/picture2.png)
+
+**1﻿.3**  You will notice a similar page as shown below. Click “create cluster” to create a new cluster, or you can also choose from the already created clusters. Ensure that you choose a cluster that does not have Istio pre-deployed, since this exercise will deploy SPIRE and Istio together.
+
+![](/img/picture4.png)
+
+**1.4**	 After clicking create cluster, give name and description to your cluster and type of cluster. In  our case we have chosen large type.
+
+![](/img/picture5.png)
+
+**1.5**	 Obtain Kubeconfig for your cluster and launch a Web terminal to access your cluster for further steps in the document. 
+
+From the **Containers** main page, click **Launch Service Console** to launch the HPE Ezmeral Runtime Enterprise. Open Kubectl, which allows you to enter commands to communicate with your cluster.
+
+If Kubectl is not installed, download Kubectl and the HPE Kubectl Plugin from the **Dashboard**.
+
+For more information, see** [Dashboard - Kubernetes Tenant/Project Administrator](https://docs.containerplatform.hpe.com/55/reference/kubernetes/tenant-project-administration/Dashboard__Kubernetes_TenantProject_Administrator.html)** in the HPE Ezmeral Runtime Enterprise documentation.
+
+### Step 2: Install SPIRE
+
+Get the quickstart yaml file using **[this link](https://raw.githubusercontent.com/cxteamtrials/caas-trials-content/main/services/spire/spire-quickstart.yaml)** and copy that into your cluster and apply it by using kubectl.
+
+```shellsession
+kubectl apply -f spire-quickstart.yaml
+```
+
+This will install SPIRE into your cluster, along with two additional components: the SPIFFE CSI Driver and the SPIRE Kubernetes **Controller manager** which facilitate the registration of workloads and establishment of federation relationships.
```

Comment:
So, a question here... does the Controller manager facilitate the registration of workloads and establishment of federation relationships? Do both the SPIFFE CSI Driver andController manager to this together? Or do these two things do the facilitation of the workload registration and establish the federation relationships respectively? I think you mean the last one, in which case you would need to write it as follows to clarify:
"...the SPIFFE CSI DRiver and the SPIRE Kubernetes Controller manager. The former facilitates the registration of workloads and the latter facilitates the establishment of federation relationships.

Reply:
Controller manager facilitate the registration and establishment of federation relationship.